app/src/main/res/layout/dialog_download.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:padding="@dimen/padding_quote">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/request_download"
+        android:layout_margin="@dimen/padding_quote"
+        android:layout_alignParentTop="true"
+        android:textColor="@color/color_black"
+        />
```

Comment:
Bad formatting.

Reply:
Can you please explain this in detail?

Reply:
`android:textColor="@color/color_black" />` instead of
```
android:textColor="@color/color_black"
/>
```